db/seeds.rb

```diff
@@ -1,7 +1,20 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+user = User.create([{ user_name: "Donard"}])
+
+user_details = UserDetail.create(
+  full_name: "Donard Golaj", 
+  avatar: "string", 
+  birthday: "11/11/11", 
+  user_id: 1)
+
+
+  meetings = Meeting.create(
+    title: "title",
+    description: "description",
+    color: "color",
+    image: "image",
+    link: "link",
+    meeting_type_id: 2,
+    user_id: 4
+  )
+
+  meetings.save
```

Comment:
The Data Seeds are based on your local, It may cause some issues in other teams' locals.
We don't have the code for creating a user with  id 4 and meeting_type with id 2 but we have the seed for creating Meeting with: `meeting_type_id: 2, user_id: 4`
Try to make the seed file a general file implementable on a fresh empty DB

Reply:
Thank you for the feedback! I fix the problem and added new data. 👍